features/support/vagrant_helpers.rb

```diff
@@ -17,17 +19,18 @@ class VagrantSSHCommandError < RuntimeError; end
   def vagrant_cli_command(command)
     puts "[vagrant] #{command}"
     Dir.chdir(VAGRANT_ROOT) do
-      `#{VAGRANT_BIN} #{command} 2>&1`.split("\n").each do |line|
-        puts "[vagrant] #{line}"
-      end
+      @stdout, @stderr, @status = Open3.capture3("#{VAGRANT_BIN} #{command}")
     end
-    $CHILD_STATUS
+
+    (@stdout + @stderr).split("\n").each { |line| puts "[vagrant] #{line}" }
+
+    @status
```

Comment:
Nice improvement to use `Open3`. 👍 

I'd like to see `vagrant_cli_command` return `stdout, stderr, status` rather than set instance variables. This will make the code easier to follow instead of relying on side-effects.